quartz_solar_forecast/utils/sentry_logging.py

```diff
@@ -0,0 +1,42 @@
+""" Log usage of this package to Sentry """
+
+import sentry_sdk
+from quartz_solar_forecast.pydantic_models import PVSite
+
+import os
+
+quartz_solar_forecast_logging = bool(os.getenv("QUARTZ_SOLAR_FORECAST_LOGGING", True))
```

Comment:
I've done some tests locally and this should be:
`quartz_solar_forecast_logging = os.getenv("QUARTZ_SOLAR_FORECAST_LOGGING", "True").lower() != "false"
`

This can handle the upper and lower case of False and behaves as expected. The current method returns True for all cases

Reply:
thanks Zak